modules/ROOT/pages/relnotes-java-sdk.adoc

```diff
@@ -31,6 +31,61 @@ cluster management. The other dependency is RxJava which is both used
 internally and exposed as a first-class API to the user in order to
 perform scalable and performant asynchronous, reactive operations.
 
+== Version 2.7.0 (2 October 2018)
+
+Version 2.7.0 is the first stable release of the 2.7 series, bringing new
```

Comment:
I'm sure this is consistent with the past, but it makes it sound like we have unstable releases, like we do even and odd.  For the future, maybe this is something we can improve @RichardSmedley ?